include/torch-mlir/Dialect/Torch/IR/TorchOps.td

```diff
@@ -953,6 +953,24 @@ def Torch_PseudoAtenBernoulliFloatOp: Torch_Op<"pseudo.aten.bernoulli.float", [
   let assemblyFormat = "$self `,` $p `,` $generator attr-dict `:` type($self) `,` type($p) `,` type($generator) `->` type($result)";
 }
 
+// The corresponding without underscore variant for `torch.aten.bernoulli_.Tensor`
```

Comment:
@cathyzhyi Shouldn't we have another `.td` file for `pseudoAtenOps`. This will also act as a clear separation from core torch ops and pseudo ops.

Reply:
I can handle that in a subsequent PR. Thanks!